common/src/main/java/com/sequenceiq/cloudbreak/util/DatabaseUtil.java

```diff
@@ -24,4 +24,9 @@ public static void createSchemaIfNeeded(String dbType, String dbAddress, String
             }
         }
     }
+
+    public static String encapsulateContentForLikelinessQuery(String content) {
```

Comment:
maybe I am wrong but I can't find any usage

Reply:
you are right, removed :)